### ISSUE TITLE
ICE in solidity::frontend::ExpressionCompiler::visit(const solidity::frontend::MemberAccess &) [unsupported member function]

### ISSUE THREAD

**Issue by agroce** [ethereum/solidity#8365]
## Description

This contract:
```solidity
contract A {
    function f() external {}}
contract B {
    function g() external {
        A.f;
    }
}
```

produces:

```
Internal compiler error during compilation:
/home/user/solidity/libsolidity/codegen/ExpressionCompiler.cpp(1342): Throw in function virtual bool solidity::frontend::ExpressionCompiler::visit(const solidity::frontend::MemberAccess &)
Dynamic exception type: boost::exception_detail::clone_impl<solidity::langutil::InternalCompilerError>
std::exception::what: unsupported member function
[solidity::util::tag_comment*] = unsupported member function
```

when run with solc

On master, using AFL fuzzing.  Another discovery using https://github.com/agroce/afl-compiler-fuzzer, now with a few more mutation operators

## Environment

- Compiler version:  0.6.3-develop.2020.2.10+commit.e8eb1f2d.Linux.clang
- Target EVM version (as per compiler settings): N/A
- Framework/IDE (e.g. Truffle or Remix): N/A
- EVM execution environment / backend / blockchain client: N/A
- Operating system: Ubuntu 18.04 in docker

## Steps to Reproduce

Above shows pretty clearly, I think.

**Comment by bshastry** [comment]
@agroce Thank you for the report. Could you please tell us the command line invocation you used to reproduce this issue?

I tried to repro using `solc <file.sol>` on the latest `develop` commit (`44bcff42f`) but could not.

**Comment by agroce** [comment]
It was solc (compiled with AFL instrumentation, maybe that matters...) on 0.6.3-develop.2020.2.10+commit.e8eb1f2d.Linux.clang

**Comment by agroce** [comment]
Yes:

```
user@042f9c682794:~/solidity/test$ cat f.sol
contract A {
    function f() external {}}
contract B {
    function g() external {
        A.f;
    }
}user@042f9c682794:~/solidity/test$ ../build/solc/solc f.sol 
Internal compiler error during compilation:
/home/user/solidity/libsolidity/codegen/ExpressionCompiler.cpp(1342): Throw in function virtual bool solidity::frontend::ExpressionCompiler::visit(const solidity::frontend::MemberAccess &)
Dynamic exception type: boost::exception_detail::clone_impl<solidity::langutil::InternalCompilerError>
std::exception::what: unsupported member function
[solidity::util::tag_comment*] = unsupported member function
user@042f9c682794:~/solidity/test$ ../build/solc/solc --version
solc, the solidity compiler commandline interface
Version: 0.6.3-develop.2020.2.10+commit.e8eb1f2d.Linux.clang
```

**Comment by axic** [comment]
Not reproducible anymore:
```
Warning: Statement has no effect.
 --> 8365.sol:5:9:
  |
5 |         A.f;
  |         ^^^

Warning: Function state mutability can be restricted to pure
 --> 8365.sol:4:5:
  |
4 |     function g() external {
  |     ^ (Relevant source part starts here and spans across multiple lines).

```

Should we try to find the PR which fixed this?

**Comment by chriseth** [comment]
Fixed in 0.6.3.
